### ISSUE TITLE
impossible to interact with pawns and noone tends to wounded

### ISSUE THREAD

**Issue by pokerfacethe14th** [FluffierThanThou/Pharmacist#42]
my colonist got injured and sat down in a medical bed like he should, but my doctor never tended to him, not even when i locked them in the same room together with the necessary meds. i cant right click to force pawn interaction, not only when theyre injured but in any situation right clicking on a pawn causes a red error to appear with no menu. it has made the game unplayable as pawns cant be tended or manually told to interact with other pawns.

**Steps to reproduce**  

there's no prequisite to making it happen, it happens by default on any new or existing save with pharmacist installed

**Other mods**  
i disabled every other mod except for harmony and hugslib to test it(i also had royalty activated)
**Output log**  
https://gist.github.com/129c24e3ea2035d6aced3c9c31a3a07b

**Save game**  
https://1drv.ms/u/s!AgCxkGMfG2Mqge5TiA01BaMVwbshZw?e=6ZwJgp

**Screenshots**  

**Environment**  
windows 10
x64
 
**Game**
64 bits
steam
1.3

**Comment by megamurmulis** [comment]
Same issue (only [`Harmony, Core, Royalty, Pharmacist`] enabled) = no context menu
And with Dev mod enabled - throws error:
```
Root level exception in OnGUI(): System.InvalidOperationException: Sequence contains no elements
  at Verse.GenCollection.MaxBy[TSource,TKey] (System.Collections.Generic.IEnumerable`1[T] source, System.Func`2[T,TResult] selector, System.Collections.Generic.IComparer`1[T] comparer) [0x00039] in <00638943358e4807bd44cd7bc8f572da>:0 
  at Verse.GenCollection.MaxBy[TSource,TKey] (System.Collections.Generic.IEnumerable`1[T] source, System.Func`2[T,TResult] selector) [0x00007] in <00638943358e4807bd44cd7bc8f572da>:0 
  at Pharmacist.HealthAIUtility_FindBestMedicine.Prefix (Verse.Pawn healer, Verse.Pawn patient, System.Boolean onlyUseInventory, Verse.Thing& __result) [0x00057] in <60f543b514a84700bb9fdd05e7dc9499>:0 
  at (wrapper dynamic-method) RimWorld.HealthAIUtility.RimWorld.HealthAIUtility.FindBestMedicine_Patch1(Verse.Pawn,Verse.Pawn,bool)
  at RimWorld.WorkGiver_Tend.JobOnThing (Verse.Pawn pawn, Verse.Thing t, System.Boolean forced) [0x00007] in <00638943358e4807bd44cd7bc8f572da>:0 
  at RimWorld.FloatMenuMakerMap.AddJobGiverWorkOrders (UnityEngine.Vector3 clickPos, Verse.Pawn pawn, System.Collections.Generic.List`1[T] opts, System.Boolean drafted) [0x001ba] in <00638943358e4807bd44cd7bc8f572da>:0 
  at RimWorld.FloatMenuMakerMap.AddUndraftedOrders (UnityEngine.Vector3 clickPos, Verse.Pawn pawn, System.Collections.Generic.List`1[T] opts) [0x00085] in <00638943358e4807bd44cd7bc8f572da>:0 
  at RimWorld.FloatMenuMakerMap.ChoicesAtFor (UnityEngine.Vector3 clickPos, Verse.Pawn pawn, System.Boolean suppressAutoTakeableGoto) [0x000b5] in <00638943358e4807bd44cd7bc8f572da>:0 
  at RimWorld.FloatMenuMakerMap.TryMakeFloatMenu (Verse.Pawn pawn) [0x000a1] in <00638943358e4807bd44cd7bc8f572da>:0 
  at RimWorld.Selector.HandleMapClicks () [0x000a9] in <00638943358e4807bd44cd7bc8f572da>:0 
  at RimWorld.Selector.SelectorOnGUI () [0x00000] in <00638943358e4807bd44cd7bc8f572da>:0 
  at RimWorld.MapInterface.HandleLowPriorityInput () [0x0000f] in <00638943358e4807bd44cd7bc8f572da>:0 
  at RimWorld.UIRoot_Play.UIRootOnGUI () [0x000dd] in <00638943358e4807bd44cd7bc8f572da>:0 
  at Verse.Root.OnGUI () [0x0003d] in <00638943358e4807bd44cd7bc8f572da>:0 
UnityEngine.StackTraceUtility:ExtractStackTrace ()
Verse.Log:Error (string)
Verse.Root:OnGUI ()
```

Temporary workaround (from  https://github.com/fluffy-mods/Pharmacist/pull/41 ) => no error/issue if the pawn doing tending picks up the medicine first..

**Comment by Arcalane** [comment]
I've heard of a couple other people with this issue and I'm experiencing it as well. This *needs* fixing ASAP - if raiders-turned-prisoners have bloodloss (and they almost always do) they'll never be treated properly and bleed out. Likewise heart attacks and certain other conditions are pretty much guaranteed to be lethal without treatment.

**Comment by Karel-Kroeze** [comment]
fixed with 02d13af, thanks for reporting (and sorry for leaving this out there for so long!)
